docs/global-components/qr-color-table.vue

```diff
@@ -26,10 +26,12 @@ const rows = computed(() => props.list ?? data[dataTitle.value])
         <td v-if="desc === null" colspan="2">Unsupported</td>
         <template v-else>
           <td :class="{ 's-text-inverted': /^s-bg-primary-/.test(cls) }">
-            <div :class="[cls, { 's-bg-primary': /^s-text-inverted/.test(cls) }]" class="border w-64 px-8">Text</div>
+            <div :class="[cls, { 's-bg-primary': /^s-text-inverted/.test(cls) },{ 'border': /border|icon|text/.test(cls) } ]" class="w-64 px-8">
```

Comment:
Do we want border for anything but the background colours?
```suggestion
            <div :class="[cls, { 's-bg-primary': /^s-text-inverted/.test(cls) },{ 'border': !/^s-bg/.test(cls) } ]" class="w-64 px-8">
```

Reply:
Makes it so much cleaner, thanks! Fixed in [863e945](https://github.com/warp-ds/css-docs/pull/75/commits/863e94577a8906f767ec9b5ee08e55ba6704c387)